src/components/pipelines-overview/list-pages/PipelineRunsForRepositoriesRow.tsx

```diff
@@ -1,23 +1,42 @@
 import * as React from 'react';
-import { mainDataType } from '../utils';
-import { RowProps } from '@openshift-console/dynamic-plugin-sdk';
+import { SummaryProps, getReferenceForModel } from '../utils';
+import { ResourceLink, RowProps, useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
+import { formatTime, formatTimeLastRunTime } from '../dateTime';
+import { ALL_NAMESPACES_KEY } from '../../../consts';
+import { RepositoryModel } from '../../../models';
 
-const tableColumnClasses = ['', '', '', '', '', '', ''];
+const tableColumnClasses = [
+  '', //name
+  '', //namespace
+  'pf-m-hidden pf-m-visible-on-md', //total plr
+  'pf-m-hidden pf-m-visible-on-md', //total duration
+  'pf-m-hidden pf-m-visible-on-xl', //avg duration
+  'pf-m-hidden pf-m-visible-on-xl', //success rate
+  'pf-m-hidden pf-m-visible-on-xl' //last run time
+];
+const repositoryReference = getReferenceForModel(RepositoryModel);
 
-const PipelineRunsForRepositoriesRow: React.FC<RowProps<mainDataType>> = ({
+const PipelineRunsForRepositoriesRow: React.FC<RowProps<SummaryProps>> = ({
   obj,
 }) => {
+  const [activeNamespace] = useActiveNamespace();
+  
   return (
     <>
-      <td className={tableColumnClasses[0]}>{obj.repoName}</td>
-      <td className={tableColumnClasses[1]}>{obj.projectName}</td>
+      <td className={tableColumnClasses[0]}>
+        <ResourceLink
+      kind={repositoryReference}
+      name={obj?.group_value.split('/')[1]}
+      namespace={obj?.group_value.split('/')[0]}
+      /></td>
+      {activeNamespace === ALL_NAMESPACES_KEY && <td className={tableColumnClasses[1]}>{obj?.group_value?.split('/')[0]}</td>}
```

Comment:
Can you also create a resource link for the namespace as well? so that it will be consistent with other lists in the console.